spec/rspec/core/configuration_spec.rb

```diff
@@ -36,6 +36,24 @@ module RSpec::Core
       end
     end
 
+    describe 'Corrupt examples.txt file' do
+      before do
+        RSpec.configure do |c|
+          c.example_status_persistence_file_path = "spec/examples-corrupted.txt"
+        end
+      end
+
+      it 'should not crash' do
+        Array.new(9) do |i|
+          RSpec.describe("group") { it "example #{i}" do; end}
+        end
+        expectation = expect do
+          RSpec.world.example_groups
+        end
+        expectation.not_to raise_error
+      end
+    end
```

Comment:
#1 Thanks for the feedback. I am not super familiar with the rspec library so I just tried my best to put it where it's already kind of testing it.

#2 is very good for sure. My main concern was just that it didn't crash but I suppose that it should be unknown, not just nil. That might make me change the code some more

#3 Ok thanks! I'll try my best to get that working